.github/workflows/release.yml

```
@@ -0,0 +1,92 @@
+name: Create GitHub Release
+
+env:
+  RUNTIME_DEPENDENCIES: '[ libudev1, libevdev2, libglib2.0-0, libgee-0.8-2, upower ]'
```

```suggestion
  RUNTIME_DEPENDENCIES: '[ libudev1, libevdev2, libglib2.0-0, libgee-0.8-2 ]'
```

UPower is a daemon that evdevhook2 (optionally) communicates with over D-Bus. Do not include it.

---

Done.